src/scripts/main.js

```diff
@@ -1,3 +1,17 @@
 'use strict';
 
-// write your code here
+const table = document.querySelector('table');
+
+table.rows[0].cells[3].insertAdjacentHTML('afterend', `
+  <th>${table.rows[0].cells[1].innerHTML}</th>
+  `);
+
+table.rows[13].cells[3].insertAdjacentHTML('afterend', `
+  <th>${table.rows[13].cells[1].innerHTML}</th>
+  `);
```

Comment:
```suggestion
```

You can easily get rid of these redundant lines of code. All you need you already have in your `for` loop. Just use `cloneNode` instead of `insertAdjacentHTML`.
No need to repeat yourself three times. You can just iteratr through ALL `tr`s and clone their needed cells